Makefile

```diff
@@ -151,3 +152,7 @@ doctest:
 	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
+
+spelling:
```

Comment:
we should also add a spelling task to `make.bat`, to make things work on Windows too.